src/frontend/js/components/PurchaseButton/index.spec.tsx

```diff
@@ -181,7 +181,7 @@ describe('PurchaseButton', () => {
     expect(button).toBeDisabled();
 
     // Further, a message is displayed to explain why the CTA is disabled
-    screen.findByText('This product is full, is currently not available for sale');
+    await screen.findByText('There are no more places available for this product.');
```

Comment:
We've an other one in the same file: 
```js
$ git grep -A 3 '  screen.find'
src/frontend/js/components/PurchaseButton/index.spec.tsx:    screen.findByText('This product is full, is currently not available for sale');
src/frontend/js/components/PurchaseButton/index.spec.tsx-  });
src/frontend/js/components/PurchaseButton/index.spec.tsx-
src/frontend/js/components/PurchaseButton/index.spec.tsx-  it.each([
--
src/frontend/js/components/PurchaseButton/index.spec.tsx:    screen.findByText(
src/frontend/js/components/PurchaseButton/index.spec.tsx-      'At least one course has no course runs, this product is not currently available for sale',
src/frontend/js/components/PurchaseButton/index.spec.tsx-    );
src/frontend/js/components/PurchaseButton/index.spec.tsx-  });

```

Reply:
Good catch, this is also a false positive ..